h/h_api/model/base.py

```diff
@@ -0,0 +1,31 @@
+class Model:
+    """Represents an object backed by a plain data structure.
+
+    For compatibility with JSON serialisation it's important that the inner
+    data structure not contain anything which cannot be serialised. This is
+    the responsibility of the implementer.
+    """
```

Comment:
Not too exciting yet, but we're going to wire these up to validators pretty soon